metadata/cadastre/utah-parcels

```diff
@@ -0,0 +1,75 @@
+# Title
+
+Utah Basic Parcels
+
+## ID
+
+cbe4da2a-3edb-4a4a-b082-60f9371c8006
+
+## Brief Summary
+
+Database representing parcel boundaries for all 29 counties in Utah that includes basic information such as parcel address and generalized land ownership.
```

Comment:
**nitpick**: database means something different than what this is. 

Here's a suggestion that tries to explain what a parcel is and removes the database term. It's not great but it should help.

```suggestion
Parcel, specific areas or plots of land, boundaries for all 29 counties in Utah that include information such as parcel address and generalized ownership.
```

Reply:
I approve this suggestion

Reply:
I've gone in and changed the wording and removed the term database everywhere I saw it!